From be370454b3e58428c10eed594430a65abc71d643 Mon Sep 17 00:00:00 2001
Message-ID: <be370454b3e58428c10eed594430a65abc71d643.1701297273.git.sharlatanus@gmail.com>
In-Reply-To: <123a0b9b76c24c29184c527dabb652384f7be762.1701297273.git.sharlatanus@gmail.com>
References: <123a0b9b76c24c29184c527dabb652384f7be762.1701297273.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Sun, 29 Oct 2023 15:11:55 +0000
Subject: [PATCH v3 38/66] gnu: go-github-com-opentracing-opentracing-go: Move
 to (gnu packages golang-web).

* gnu/packages/golang.scm (go-github-com-opentracing-opentracing-go): Move from
here...
* gnu/packages/golang-web.scm: ...to here.

Change-Id: I87e615589cdb038e2b633ecd63194c81d5864a5b
---
 gnu/packages/golang-web.scm | 25 +++++++++++++++++++++++++
 gnu/packages/golang.scm     | 24 ------------------------
 2 files changed, 25 insertions(+), 24 deletions(-)

diff --git a/gnu/packages/golang-web.scm b/gnu/packages/golang-web.scm
index caccb126bc..d021be005f 100644
--- a/gnu/packages/golang-web.scm
+++ b/gnu/packages/golang-web.scm
@@ -5,6 +5,7 @@
 ;;; Copyright © 2020 Nicolas Goaziou <mail@nicolasgoaziou.fr>
 ;;; Copyright © 2020 Oleg Pykhalov <go.wigust@gmail.com>
 ;;; Copyright © 2020 Ryan Prior <rprior@protonmail.com>
+;;; Copyright © 2021 Raghav Gururajan <rg@raghavgururajan.name>
 ;;; Copyright © 2021 Sarah Morgensen <iskarian@mgsn.dev>
 ;;; Copyright © 2021, 2022 Efraim Flashner <efraim@flashner.co.il>
 ;;; Copyright © 2022 Giacomo Leidi <goodoldpaul@autistici.org>
@@ -703,6 +704,30 @@ (define-public go-github-com-microcosm-cc-bluemonday
     (description "@code{bluemonday} is a HTML sanitizer implemented in Go.")
     (license license:bsd-3)))
 
+(define-public go-github-com-opentracing-opentracing-go
+  (package
+    (name "go-github-com-opentracing-opentracing-go")
+    (version "1.2.0")
+    (source
+     (origin
+       (method git-fetch)
+       (uri
+        (git-reference
+         (url "https://github.com/opentracing/opentracing-go")
+         (commit (string-append "v" version))))
+       (file-name (git-file-name name version))
+       (sha256
+        (base32 "04rgdwl29kimp2wnm4dycnzp7941hvpj6wym85x23c6fclacm94h"))))
+    (build-system go-build-system)
+    (arguments
+     `(#:import-path "github.com/opentracing/opentracing-go"))
+    (native-inputs
+     (list go-github-com-stretchr-testify))
+    (home-page "https://github.com/opentracing/opentracing-go")
+    (synopsis "OpenTracing API for Go")
+    (description "OpenTracing-Go is a Go implementation of the OpenTracing API.")
+    (license license:asl2.0)))
+
 (define-public go-github-com-puerkitobio-goquery
   (package
     (name "go-github-com-puerkitobio-goquery")
diff --git a/gnu/packages/golang.scm b/gnu/packages/golang.scm
index 310c2ba8f4..69efe7c6bf 100644
--- a/gnu/packages/golang.scm
+++ b/gnu/packages/golang.scm
@@ -1798,30 +1798,6 @@ (define-public go-github-com-mufti1-interconv
       (description "InterConv converts interfaces into any data type.")
       (license license:expat))))
 
-(define-public go-github-com-opentracing-opentracing-go
-  (package
-    (name "go-github-com-opentracing-opentracing-go")
-    (version "1.2.0")
-    (source
-     (origin
-       (method git-fetch)
-       (uri
-        (git-reference
-         (url "https://github.com/opentracing/opentracing-go")
-         (commit (string-append "v" version))))
-       (file-name (git-file-name name version))
-       (sha256
-        (base32 "04rgdwl29kimp2wnm4dycnzp7941hvpj6wym85x23c6fclacm94h"))))
-    (build-system go-build-system)
-    (arguments
-     `(#:import-path "github.com/opentracing/opentracing-go"))
-    (native-inputs
-     (list go-github-com-stretchr-testify))
-    (home-page "https://github.com/opentracing/opentracing-go")
-    (synopsis "OpenTracing API for Go")
-    (description "OpenTracing-Go is a Go implementation of the OpenTracing API.")
-    (license license:asl2.0)))
-
 (define-public go-github-com-operatorfoundation-monolith-go
   (package
     (name "go-github-com-operatorfoundation-monolith-go")
-- 
2.41.0

